en/gps_compass/septentrio.md

```diff
@@ -28,19 +28,23 @@ Certain receivers are recommended for autopilot applications because of their ph
 - RTK positioning using corrections over MAVLink from a ground control station
 - Dual-antenna heading with mosaic-H-based receivers
 - Moving base heading with two single-antenna receivers
-- SBF logging to internal storage on the receiver
+- Flexible SBF logging to internal storage on the receiver
 - Logging of all communication with the receiver to internal storage on the flight controller
 - Automatic baud rate and port detection
-- Automatic, parameterized configuration for all features
+- Automatic, parameterized configuration for all features (optional)
 - Status and health monitoring from a MAVLink console
 
 ## Quick Start
 
 For the physical connection between Septentrio GNSS receivers and flight controllers, have a look at the hardware-specific guides for [mosaic-go receivers](septentrio_mosaic-go.md) and [AsteRx-m3 receivers with a robotics interface board](../gps_compass/septentrio_asterx-rib.md).
 
 A few things are required for PX4 to support Septentrio receivers.
-First of all, the firmware you load on your flight controller needs to contain the Septentrio driver.
-This should be the case for most flight controller boards from PX4 v1.15.
+First of all, the firmware you load on your flight controller needs to contain the Septentrio driver.  
+This is currently the case for the following flight controllers:
```

Comment:
From what version of PX4 is this true on these FCs?

Is there a roadmap for support on other vehicles?

Is it possible to  build with support on other boards, and if so how?